bootstrap.sh

```diff
@@ -17,4 +17,4 @@ kubectl apply -f .infrastructure/app/deployment.yml
 
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
-# kubectl apply -f .infrastructure/ingress/ingress.yml
+kubectl apply -f .infrastructure/ingress/ingress.yml
```

Comment:
The `ingress.yml` file is missing from the `.infrastructure/ingress/` directory. Please ensure this file is present to manage ingress traffic properly.